druid-shell/src/platform/x11/clipboard.rs

```diff
@@ -32,26 +75,193 @@ impl Clipboard {
     }
 
     pub fn get_string(&self) -> Option<String> {
-        // TODO(x11/clipboard): implement Clipboard::get_string
-        warn!("Clipboard::set_string is currently unimplemented for X11 platforms.");
+        for target in STRING_TARGETS.iter() {
+            if let Some(data) = self.get_format(target) {
+                return String::from_utf8(data).ok();
+            }
+        }
         None
```

Comment:
I couldn't resist :sweat_smile: . But feel free to ignore this
```suggestion
        STRING_TARGETS
            .iter()
            .find_map(|target| self.get_format(target))
            .and_then(|data| String::from_utf8(data).ok())
```